packages/client3/components/design-system/input/dollar-input.tsx

```diff
@@ -45,34 +45,36 @@ export function DollarInput({
   });
 
   return (
-    <MaskedInput
-      mask={mask}
-      blocks={{
-        amount: {
-          mask: Number,
-          thousandsSeparator: ",",
-          lazy: false,
-          scale: USDC_DECIMALS,
-          radix: ".",
-        },
-      }}
-      // @ts-expect-error unmask isn't typed properly in IMaskMixin for some reason
-      unmask
-      onAccept={onChange}
-      lazy={false}
-      decoration={
-        onMaxClick ? (
-          <button
-            type="button"
-            onClick={onMaxClick}
-            className="block rounded-md border border-sky-500 p-2 text-[10px] uppercase leading-none"
-          >
-            Max
-          </button>
-        ) : undefined
-      }
-      {...rest}
-      {...controllerField}
-    />
+    <div>
+      <MaskedInput
+        mask={mask}
+        blocks={{
+          amount: {
+            mask: Number,
+            thousandsSeparator: ",",
+            lazy: false,
+            scale: USDC_DECIMALS,
+            radix: ".",
+          },
+        }}
+        // @ts-expect-error unmask isn't typed properly in IMaskMixin for some reason
+        unmask
+        onAccept={onChange}
+        lazy={false}
+        decoration={
+          onMaxClick ? (
+            <button
+              type="button"
+              onClick={onMaxClick}
+              className=" block rounded-md p-2 text-[10px] uppercase leading-none"
+            >
+              Max
+            </button>
```

Comment:
I don't see any requirement for the max button on the design system

Reply:
Sure I can comment it out and leave it for reference.